ansible-scylla-monitoring/tasks/common.yml

```diff
@@ -71,7 +90,7 @@
 - name: set prometheus rules file from preset file
   copy:
     src: "{{ prometheus_rules_file }}"
-    dest: "{{ scylla_monitoring_config_path }}/prometheus.rules.yml"
+    dest: "{{ scylla_monitoring_prom_rules_path }}/userdefined.rules.yml"
```

Comment:
What about older Monitoring versions?
Wouldn't this effectively break their support?

Reply:
nope, this is only in case you want to amend/add new rules not present in old monitoring
HOWEVER it breaks old behaviour, where you could completely wipe out what monitoring provides by your own config file (do we even want to support that?)